server/utils/services/notificationService.js

```diff
@@ -0,0 +1,89 @@
+import models from '../../database/models';
+import tokenService from './tokenService';
+import notificationServiceHelper from '../helpers/notificationServiceHelper';
+
+const {
+  includeNotifn,
+  buildNotifnEndpt,
+  buildNotifnList,
+  buildNotifnMsg,
+} = notificationServiceHelper;
+
+const { Notification, UserNotifications } = models;
+
+/**
+ * Call this function to notify the users associated to a notification
+ * when it is generated
```

Comment:
Do you think that a better description would be `Notifies users associated to a when it is generated`